modules/admin-ui-frontend/app/scripts/shared/partials/wizards/new-series.html

```diff
@@ -262,7 +262,7 @@
                   </tr>
                   <tr ng-if="!wizard.step.editing">
                     <td colspan="3">
-                      <a ng-click="wizard.step.addChild()" translate="EVENTS.SERIES.NEW.TOBIRA.ADD_SUBPAGE"><!-- Add subpage --></a>
+                      <a ng-click="wizard.step.addChild()">+ {{'EVENTS.SERIES.NEW.TOBIRA.ADD_SUBPAGE' | translate}}</a>
```

Comment:
It's probably just my lack of knowledge, but what is the `+ ` doing there?

Reply:
That was part of the feedback we got. Add a "+" here for consistency with the rest of the Admin UI. (See for example "+ New policy" in the ACL editor.)

Reply:
Ohh ye I'm stupid, makes sense.